### PR TITLE
Added basic SVGOverlay to expose SVG DOM element

### PR DESCRIPTION
**Pull request by Zsavajji**
Adds a new Overlay class to display SVG elements and possibly retrieve the DOM element reference, to enable interactive SVGs into Leaflet maps.
Based onto the advice given into https://github.com/Leaflet/Leaflet/pull/5757.

### CODE REVIEW

File: src/layer/SVGOverlay.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	// Returns the instance of [`SVGElement`](https://developer.mozilla.org/docs/Web/API/SVGElement)
+	// used by this overlay.
+});
+
+
+// @factory L.svgOverlay(video: String|Array|SVGElement, bounds: LatLngBounds, options?: SVGOverlay options)
+// Instantiates an image overlay object given the URL of the video (or array of URLs, or even a video element) and the
+// geographical bounds it is tied to.
+
+export function svgOverlay(video, bounds, options) {
```
Comment (IvanSanchez) [inline src/layer/SVGOverlay.js]: Don't call the first parameter `video`.

Comment (Zsavajji) [reply src/layer/SVGOverlay.js]: You are right, i based this on the VideoOverlay, and missed that param. I also have an improvement to make, tomorrow i'll check that out.

**Comment by Zsavajji** [comment]
Tried to improve it by loading an element with XHR, but i don't have much time now to wrap it all around.
I think it's better to keep it simple by now.

**Comment by Zsavajji** [comment]
Any updates on this @IvanSanchez ?

**Comment by espensgr** [comment]
Any progress on this? would love to get this feature for a project i'm working on

**Comment by IvanSanchez** [comment]
> Checked it out locally - looks good +1

In it goes, then.

**Pull request merged**